src/script/E2EIdentity/DelayTimer/DelayTimer.ts

```diff
@@ -32,42 +32,21 @@ interface CreateGracePeriodTimerParams {
   delayPeriodExpiredCallback: () => void;
 }
 
-class DelayTimerService {
-  private static instance: DelayTimerService | null = null;
+export class DelayTimerService {
   private gracePeriodInMS: number;
   private gracePeriodExpiredCallback: () => void;
   private delayPeriodExpiredCallback: () => void;
   private readonly logger = logdown('@wireapp/core/DelayTimer');
   private delayPeriodTimerKey: string = 'E2EIdentity_DelayTimer';
   private gracePeriodTimerKey: string = 'E2EIdentity_GracePeriodTimer';
 
-  private constructor({
-    gracePeriodInMS,
-    gracePeriodExpiredCallback,
-    delayPeriodExpiredCallback,
-  }: CreateGracePeriodTimerParams) {
+  constructor({gracePeriodInMS, gracePeriodExpiredCallback, delayPeriodExpiredCallback}: CreateGracePeriodTimerParams) {
     this.gracePeriodInMS = gracePeriodInMS;
     this.gracePeriodExpiredCallback = gracePeriodExpiredCallback;
     this.delayPeriodExpiredCallback = delayPeriodExpiredCallback;
     this.initialize();
   }
 
-  /**
-   * Get the singleton instance of GracePeriodTimer or create a new one
-   * For the first time, params are required to create the instance
-   * @param params The params to create the grace period timer
-   * @returns The singleton instance of GracePeriodTimer
-   */
-  public static getInstance(params?: CreateGracePeriodTimerParams) {
-    if (!DelayTimerService.instance) {
-      if (!params) {
-        throw new Error('DelayTimerService is not initialized. Please call getInstance with params.');
-      }
-      DelayTimerService.instance = new DelayTimerService(params);
-    }
-    return DelayTimerService.instance;
-  }
```

Comment:
It was never needed that this class is a singleton. 
It simplifies the API to make it a regular instantiable class